ui_framework/components/table/table_row_check_box_cell.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { KuiTableRowCell } from './table_row_cell';
+import { KuiTableRowCellLiner } from './table_row_cell_liner';
+
+export const KuiTableRowCheckBoxCell = ({ onChange, isChecked, className, ...rest }) => {
+  const classes = classNames('kuiTableRowCell--checkBox', className);
+  return <KuiTableRowCell className={ classes } {...rest} >
+    <KuiTableRowCellLiner>
+      <input type="checkbox" className="kuiCheckBox" onChange={ onChange } checked={ isChecked } />
```

Comment:
I'd just apply the rules from the [HTML style guide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md) to our JSX, because they're both declarative "markup" with similar rules. There's a rule in there about multiple attributes on multiple lines.